### DISCUSSION TITLE
Project status

### DISCUSSION THREAD

**Discussion by walpolea** [Stegosource/vuetensils#153] [General]
Hey Austin (and other project contributors!),

Love the idea of Vuetensils; I wish there were more libraries out there taking this approach to components. I'm trying to do some due diligence on incorporating Vuetensils into the sites I build and just wanted to see if I could get a comment on the status and health of the project. Are there plans to continue to actively build out components, or is it in a maintenance mode, or is there a chance that this library may slowly fade away as many do?

Thanks in advance!

Best,
Andrew

**Comment by AustinGil** [comment]
Hey Andrew, thanks for the question. Vuetensils began as just a repo of copy/past components for the various projects I was working on. Through it's history, it's always mostly served my needs and while there's probably still some things I could add to it, I'm not spending quite as much time in Vue projects. With that being said, I still consider the project alive and well. I work on it as needs arise, but I don't intend on super active development in direction of things I don't personally need. I think a lot of the components are pretty much complete to where I'd like to take them.

To answer your question directly, the project is not fading away any time soon. I guess it's probably closer to maintenance mode than active development.

I've also always encouraged people to treat the source code as a copy/paste solution. Every component is very easy to pull into your own project and do with what you will. In some cases, I think that's a better approach than a library install. So if you haven't considered that, think about it.

And if you're looking for alternatives, I've been wanting to try Prime Vue for a while. They have an extensive collection of components and recently announces a styleless option https://primevue.org/

Hope that addresses your questions and concerns :)

**Reply by walpolea** [reply]
Thank you for such a quick and thorough response! Knowing you'll continue to have an eye on the project helps alleviate my concerns about any issues cropping up in the near/medium term. Primevue crossed my path a while ago but I did not know about the styleless option, so that's a great call-out. Take care and thank you again!

**Reply by AustinGil** [reply]
Oh yeah, I'm not going anywhere any time soon. But I am a strong proponent for preferring copy/paste solutions over library installs. Fewer dependencies means less bloat and less maintenance. Although it is nice to get upgrades too :)
